book.rb

```diff
@@ -0,0 +1,14 @@
+# This class represents the books
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    @rentals << Rental.new(date, self, person)
```

Comment:
- Hey, great using the `@rentals << Rental.new(date, self, person)` to append a new Rental object to the @rentals array to keep a collection of Rental objects for later use or processing, however, this would be also be achieved when you push into the @rentals array when you're adding the basic UI, in order not to repeat yourself, you should leave it as `Rental.new(date, self, person)`.

Reply:
Done, thank you for the review